### ISSUE TITLE
Pill widget too big?

### ISSUE THREAD

**Issue by pective** [PranshulGG/WeatherMaster#243]
I wanted to but the pill widget on my home page, but there's more space around the widget than there should be. Other weather apps have the pill widget that fits good. I saw that in other issues that the height is fixed in the other widgets, but i dont think the pill widget should be 3x3, it should fit in 2x3![Screenshot_20241219-092301.png](https://github.com/user-attachments/assets/b89746ab-0f05-49b2-b018-1dd3e8e6ab4a)

**Comment by Tuan1-2-3** [comment]
Yes

**Comment by pective** [comment]
any update on this?

**Comment by Joe-D01** [comment]
> any update on this?

The widget shape is weird, and making it smaller won’t fit into a 2x3 since it’s basically a square. I’m not working on the widgets right now because the background updates aren’t fixed yet—there’s no point if they don’t update automatically. It’s complicated with JS, and I haven’t found a solid fix yet.

**Comment by Joe-D01** [comment]
> any update on this?

The widget shape is weird, and making it smaller won’t fit into a 2x3 since it’s basically a square. I’m not working on the widgets right now because the background updates aren’t fixed yet—there’s no point if they don’t update automatically. It’s complicated with JS, and I haven’t found a solid fix yet.

**Comment by PranshulGG** [comment]
@Joe-D01 also the color palette

**Comment by pective** [comment]
hey, the issue still persists, im not sure if you quite understood what i meant.

the pill widget appears to be limited to a 3 vertical grid size, when it should perfectly fit in a 2 vertical size. the 1st widget in the video is the weather master widget, and the 2nd one is a different pill widget, that perfectly fits in 2x3 size. I dont know if it was just a misunderstanding or something wrong on my part, thanks!

**Comment by pective** [comment]
(forgot to attach the video) https://github.com/user-attachments/assets/d5311660-f2e0-4612-a409-a727a7a17eeb

**Comment by PranshulGG** [comment]
@pective should work now

**Comment by pective** [comment]
yup, works as intended now, thanks for the extremely fast fix!

**Comment by PranshulGG** [comment]
> yup, works as intended now, thanks for the extremely fast fix!

Yeah, it’s annoying when it’s something easy to fix, and I totally missed it lol
